### DISCUSSION TITLE
Double curtain switch Loratap

### DISCUSSION THREAD

**Discussion by goskim** [make-all/tuya-local#1759] [Q&A]
Hi , first of all unfortunately im newbie so please help me step by step :) so i have some problems with correct configuration of my double curtain switch made by Loratap, i have added device using local key and id from iot , but all options cause that my switch works like single Switch , i have found one similar topic with the almoust the same problem as mine, but i didnt understand everything, i need the instruction , thanks a lot for yours help:)

**Comment by make-all** [comment]
Is your model QCSC420W?  If not, it is likely an unsupported device.  Even if it is, you can't expect much help if you don't report the log messages.

**Reply by goskim** [reply]
So please tell me the best method to check logs, and my device is SC420W, typical the most popular double curtain Switch from Ali. Thanks for your help
